config/locales/simple_form.ar.yml

```diff
@@ -5,8 +5,15 @@ ar:
       defaults:
         avatar: ملف PNG أو GIF أو JPG. حجمه على أقصى تصدير 2MB. سيتم تصغيره إلى 400x400px
         digest: تُرسَل إليك بعد مُضيّ مدة مِن خمول نشاطك و فقط إذا ما تلقيت رسائل شخصية مباشِرة أثناء فترة غيابك مِن الشبكة
+        display_name:
+          one: <span class="name-counter">1</span> حرف باقي
+          other: <span class="name-counter">%{count}</span> حروف متبقية
```

Comment:
https://github.com/tootsuite/mastodon/issues/6673 🤔

Reply:
Hmm, it was recently changed: https://weblate.joinmastodon.org/translate/mastodon/simple_form/ar/?checksum=721104afa013e3f9#history and https://weblate.joinmastodon.org/translate/mastodon/simple_form/ar/?checksum=3edf6723cddf9edd#history
Don't know what's the best way here :thinking:

Reply:
Could it be related to https://github.com/WeblateOrg/weblate/issues/1550 ?
I can remove again to avoid bumping into the profile error

Reply:
cc @BoFFire

Reply:
These 2 strings causes a problem for the user in arabic to access the profile to change avatar and so on. I don't know how to deal with spans :(

Reply:
Related to : https://github.com/tootsuite/mastodon/issues/6673